src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface.jl

```diff
@@ -87,7 +88,7 @@ function SplitExplicitFreeSurface(grid; gravitational_acceleration = g_Earth,
     η = ZFaceField(grid, indices = (:, :, size(grid, 3)+1))
 
     return SplitExplicitFreeSurface(η, SplitExplicitState(grid), SplitExplicitAuxiliaryFields(grid),
-                                    gravitational_acceleration, settings)
+                                    convert(eltype(grid), gravitational_acceleration), settings)
```

Comment:
when is this used?
it seems like it would overwrite what the users might have prescribed with method `SplitExplicitFreeSurface(FT::DataType = Float64; gravitational_acceleration = g_Earth, kwargs...)`, no?